docs/advanced/addons/managed-dhcp.md

```diff
@@ -0,0 +1,248 @@
+
+### API
+
+The [vm-dhcp-controller](https://github.com/harvester/vm-dhcp-controller) introduces two new CRDs.
+
+- IPPool (ippl)
+- VirtualMachineNetworkConfig (vmnetcfg)
+
+#### IPPool
+
+The IPPool CRD is for users to define IP pool information. It is required to have the NetworkAttachmentDefinition (NAD) object created beforehand. Each IPPool object must be mapped to a specific NAD object.
+
+For example:
+
+```yaml
+apiVersion: network.harvesterhci.io/v1alpha1
+kind: IPPool
+metadata:
+  name: example
+  namespace: default
+spec:
+  ipv4Config:
+    serverIP: 192.168.100.2 # The DHCP server's IP address
+    cidr: 192.168.100.0/24 # The subnet information, must be in the CIDR form
+    pool:
+      start: 192.168.100.101
+      end: 192.168.100.200
+      exclude:
+      - 192.168.100.151
+      - 192.168.100.187
+    router: 192.168.100.1 # The default gateway, if any
+    dns:
+    - 1.1.1.1
+    domainName: example.com
+    domainSearch:
+    - example.com
+    ntp:
+    - pool.ntp.org
+    leaseTime: 300
+  networkName: default/example # The namespaced name of the NAD object
+```
+
+After the creation of the IPPool object, the controller reconciliation process will initialize the IP allocation module and spawn the agent Pod for the network.
+
+```shell
+$ kubectl get ippools.network example
+NAME      NETWORK           AVAILABLE   USED   REGISTERED   CACHEREADY   AGENTREADY
+example   default/example   98          0      True         True         True
+```
+
+#### VirtualMachineNetworkConfig
+
+The VirtualMachineNetworkConfig CRD resembles a **request for IP address issuance**. It is associated with NetworkAttachmentDefinition objects.
+
+A sample VirtualMachineNetworkConfig object looks like the following:
+
+```yaml
+apiVersion: network.harvesterhci.io/v1alpha1
+kind: VirtualMachineNetworkConfig
+metadata:
+  name: test-vm
+  namespace: default
+spec:
+  networkConfig:
+  - macAddress: 22:37:37:82:93:7d
+    networkName: default/example
+  vmName: test-vm
+```
+
+After the object being created, the controller tries to get unused IP addresses from the IP allocation module for each MAC address recorded. The IP-MAC mapping will then be updated in the VirtualMachineNetworkConfig object and the corresponding IPPool objects.
+
+:::note 
+In normal cases, it is unnecessary to manually create VirtualMachineNetworkConfig objects for virtual machines. vm-dhcp-controller will help create them during the VirtualMachine reconciliation process. The auto-created VirtualMachineNetworkConfig objects will be deleted along with the removal of VirtualMachine objects.
+:::
+
+### Usage
+
+1. Create a [VM Network](../../networking/harvester-network.md#create-a-vm-network) on the Harvester dashboard
+
+    ![](/img/v1.3/vm-dhcp-controller/vm-network.png)
+
+1. Create an IPPool object through the command line tool `kubectl`
```

Comment:
When creating IPPool object is not available from the WebUI, please add a note; or add an usage picture.